specification/cognitiveservices/data-plane/Personalizer/preview/v2022-09-01-preview/Personalizer.json

```diff
@@ -0,0 +1,2847 @@
+							"$ref": "#/definitions/Evaluation"
+						}
+					},
+					"default": {
+						"description": "Offline Evaluation not found.",
+						"schema": {
+							"$ref": "#/definitions/ErrorResponse"
+						}
+					}
+				},
+				"x-ms-examples": {
+					"Successful Evaluations_Get request": {
+						"$ref": "./examples/Evaluations_Get.json"
+					}
+				}
+			},
+			"delete": {
+				"tags": [
+					"Evaluations20220901Preview"
+				],
+				"summary": "Delete Evaluation.",
+				"description": "Delete the Offline Evaluation associated with the Id.",
+				"operationId": "Evaluations_Delete",
+				"consumes": [],
+				"produces": [],
+				"parameters": [
+					{
+						"name": "api-version",
+						"in": "query",
+						"required": true,
+						"type": "string",
+						"description": "Client API version."
+					},
+					{
+						"name": "evaluationId",
+						"in": "path",
+						"description": "Id of the Offline Evaluation to delete.",
+						"required": true,
+						"type": "string",
+						"maxLength": 256
+					}
+				],
+				"responses": {
+					"204": {
+						"description": "Success"
+					}
+				},
+				"x-ms-examples": {
+					"Successful Evaluations_Delete request": {
+						"$ref": "./examples/Evaluations_Delete.json"
+					}
+				}
+			}
+		},
+		"/evaluations": {
+			"get": {
+				"tags": [
+					"Evaluations20220901Preview"
+				],
+				"summary": "List Offline Evaluations.",
+				"description": "List of all Offline Evaluations.",
+				"operationId": "Evaluations_List",
+				"consumes": [],
+				"produces": [
+					"application/json"
+				],
+				"parameters": [
+					{
+						"name": "api-version",
+						"in": "query",
+						"required": true,
+						"type": "string",
+						"description": "Client API version."
+					},
+					{
+						"in": "query",
+						"name": "filter",
+						"description": "An expression to filter the evaluations against evaluations metadata. Only evaluations where the expression evaluates to true are included in the response.",
+						"type": "string"
+					},
+					{
+						"in": "query",
+						"name": "orderby",
+						"description": "The ordering. Defaults to creation time in descending order.",
+						"type": "string",
+						"default": "creationTime desc"
+					},
+					{
+						"in": "query",
+						"name": "top",
+						"description": "The maximum number of resources to return from the collection. Defaults to maximum value of integer.",
+						"type": "integer",
+						"format": "int32",
+						"default": 2147483647
+					},
+					{
+						"in": "query",
+						"name": "skip",
+						"description": "An offset into the collection of the first resource to be returned. Defaults to 0.",
+						"type": "integer",
+						"format": "int32",
+						"default": 0
+					},
+					{
+						"in": "query",
+						"name": "maxpagesize",
+						"description": "The maximum number of resources to include in a single response. Defaults to 100.",
+						"type": "integer",
+						"format": "int32",
+						"default": 100
```

Comment:
It is better to not specify a default for `maxpagesize` because you might want to change it later or even vary it based on circumstances.

Reply:
Addressed.